<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cvWorkClass</class>
 <widget class="QMainWindow" name="cvWorkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简单物体识别工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(175,213,239);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="surfBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="text">
     <string>SURF算子</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="siftBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>SIFT算子</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px dashed gray;
height:400px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="topLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px dashed gray;
height:400px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <zorder>surfBtn</zorder>
   <zorder>imageLabel</zorder>
   <zorder>siftBtn</zorder>
   <zorder>topLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>提取</string>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>训练</string>
   </property>
  </action>
  <action name="actionExtract_2">
   <property name="text">
    <string>提取</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cvwork.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_surfBtn_clicked()</slot>
  <slot>on_siftBtn_clicked()</slot>
  <slot>on_actionOpen_triggered()</slot>
  <slot>on_actionTrain_triggered()</slot>
  <slot>on_actionExtract_triggered()</slot>
 </slots>
</ui>
